### PR TITLE
add face alignment

### PR DESCRIPTION
**Pull request by greensun0830**
在mlir的sdk环境下将face_align模型转换为bmodel，芯片要求支持1684x，fp32和int8都已跑通，支持mlir和1684x

### CODE REVIEW

File: vision/detection/face_alignment/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+|--------------------|:----------------------------|
+| face_alignment (onnx) | [6.02 MB](face_alignment.onnx) |
+
+## Dataset
+
+[MobileFaceNet and MV-Softmax on MS-Celeb-1M-v1c](https://github.com/JDAI-CV/FaceX-Zoo/tree/main/data)
+
+## References
+
+[FaceX-Zoo](https://github.com/JDAI-CV/FaceX-Zoo/tree/main/face_sdk)
```
Comment (Jeffwhen) [inline vision/detection/face_alignment/README.md:21]: paper 链接

File: vision/recognition/face_alignment/cali_table
Comment (Jeffwhen) [inline vision/recognition/face_alignment/cali_table:1]: 不传。

**Review by Jeffwhen — CHANGES_REQUESTED** [review]

**Comment by XiangBin-gh** [comment]
需要附上runtime测试数据

**Comment by greensun0830** [comment]
name,shape,gops,time(ms),mac_utilization,cpu_usage,ddr_utilization
face_alignment_bm1684x_int8_asym,1x3x112x112,39.285,1233.937,0.10%,790.40%,0.02%
face_alignment_bm1684x_f32,1x3x112x112,39.285,724.765,2.71%,793.30%,0.04%
face_alignment_bm1684x_int8_sym,1x3x112x112,39.285,939.836,0.13%,794.50%,0.01%

**Comment by XiangBin-gh** [comment]
runtime实测数据需要提供编译器及驱动版本号，另外现在target不要直接写1684x

**Pull request closed without merging**